projects/Website Blocker/website_unblocker.py

```diff
@@ -0,0 +1,18 @@
+import platform
+
+if platform.system() == "Windows":
+        pathToHosts=r"C:\Windows\System32\drivers\etc\hosts"
+elif platform.system() == "Linux":
+        pathToHosts=r"~\etc\hosts"
```

Comment:
like mentioned earlier, slashes should be the other way 
Change it to `pathToHosts=r"/etc/hosts"`